components/ModEditFacilitySection.vue

```diff
@@ -225,22 +225,55 @@
             />
         </div>
         <ModHealthcareProfessionalSearchbar data-testid="mod-facility-section-doctor-search" />
-        <button
-            v-show="screenStore.activeScreen === ModerationScreen.EditFacility"
-            type="submit"
-            class="bg-currentColor text-white font-bold py-2 px-4 my-2 rounded w-56"
+        <div class="flex flex-col">
+            <span
+                v-if="moderationScreenStore.activeScreen === ModerationScreen.EditFacility"
+                class="mb-1 text-primary-text text-2xl font-bold font-sans leading-normal"
+            >
+                {{ $t('modFacilitySection.healthcareProfessionalToAdd') }}
+            </span>
+            <span
+                v-show="!facilityStore.healthProfessionalsRelationsForDisplay"
+                class="font-semibold"
+            >- {{ $t('modFacilitySection.noHPSelected') }}
```

Comment:
this `-` formatting look right?

Reply:
This is just a hardcoded - to show the text. It doesn't need to be translated